spec/system/candidate_interface/submitting/candidate_submitting_application_with_enic_interruption_spec.rb

```diff
@@ -0,0 +1,123 @@
+require 'rails_helper'
+
+RSpec.describe 'Candidate submits the application with enic interruption pages' do
+  include CandidateHelper
+  include SignInHelper
+
+  scenario 'Candidate submits an application with all applications having an enic_reason of not_needed', :js do
```

Comment:
```suggestion
  scenario 'Candidate submits an application with all applications having an enic_reason of not_needed', :js, time: mid_cycle do
```

This will fail between cycles when submission is blocked, so we need to make sure we travel to mid-cycle for it to pass all the time.